OxfordBodleian/Ullendorff/BDLeu4.xml

```diff
@@ -237,25 +271,428 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
 
                     </physDesc>
 
-                <!--    <history>
+                <history>
                         <origin>
-                            <origDate when="" evidence=""/>
+                            <origDate notBefore="1600" notAfter="1799" evidence="lettering">seventeenth–eighteenth century</origDate>
```

Comment:
I think it would be better to give no dating here, though I understand it refers to the current unit of circulation. We will probably come back to this after the revision of composite manuscripts soon